app/js/main.js

```diff
@@ -111,10 +114,24 @@ function openIndex() {
     });
 }
 
+/**
+ * Confirm prompt when animator is closed.
+ */
+win.on("close", function() {
+    confirmSet(closeAnimator, "", "Are you sure you to exit Boats Animator?");
+});
+
+function closeAnimator() {
```

Comment:
We don't call `closeAnimator()` anywhere else, so merge it into the `confirmSet()` call.